src/dc_imex.rs

```diff
@@ -260,25 +260,19 @@ pub unsafe extern "C" fn dc_render_setup_file(
                         replacement,
                     );
                     free(replacement as *mut libc::c_void);
-                    let setup_message_title: *mut libc::c_char = dc_stock_str(context, 42i32);
-                    let mut setup_message_body: *mut libc::c_char = dc_stock_str(context, 43i32);
-                    dc_str_replace(
-                        &mut setup_message_body,
-                        b"\r\x00" as *const u8 as *const libc::c_char,
-                        0 as *const libc::c_char,
-                    );
-                    dc_str_replace(
-                        &mut setup_message_body,
-                        b"\n\x00" as *const u8 as *const libc::c_char,
-                        b"<br>\x00" as *const u8 as *const libc::c_char,
-                    );
+                    let setup_message_title =
+                        CString::new(context.stock_str(StockMessage::AcSetupMsgSubject).as_ref())
+                            .unwrap();
+                    let setup_message_body = context.stock_str(StockMessage::AcSetupMsgBody);
+                    let msg_body_head: &str = setup_message_body.split('\r').next().unwrap();
+                    let msg_body_html = CString::new(msg_body_head.replace("\n", "<br>")).unwrap();
                     ret_setupfilecontent =
                         dc_mprintf(b"<!DOCTYPE html>\r\n<html>\r\n<head>\r\n<title>%s</title>\r\n</head>\r\n<body>\r\n<h1>%s</h1>\r\n<p>%s</p>\r\n<pre>\r\n%s\r\n</pre>\r\n</body>\r\n</html>\r\n\x00"
```

Comment:
Now that all these are rust strings, maybe better to switch to using `format!` instead of transforming them to cstrings?

Reply:
This is tempting indeed.  But it's the wrong PR as that's a refactor of dc_render_setup_file() rather than a refactor of stock strings, if I did that in too many places things would get out of hand.  I'll do it as a followup PR though.